node/src/cli.rs

```diff
@@ -86,6 +94,14 @@ pub struct ExportGenesisWasmCommand {
 	/// The name of the chain for that the genesis wasm file should be exported.
 	#[structopt(long)]
 	pub chain: Option<String>,
+
+	/// Number of accounts to be funded in the genesis in dev mode (default 10)
+	#[structopt(long, default_value = "10")]
+	pub accounts: u32,
+
+	/// Mnemonic from which we can derive funded accounts in the genesis
+	#[structopt(long)]
+	pub mnemonic: Option<String>,
```

Comment:
These shouldn't be necessary when exporting the wasm. The wasm is the compiled runtime logic which does not depend on the genesis accounts.